opencga-core/src/main/java/org/opencb/opencga/core/api/FieldConstants.java

```diff
@@ -291,7 +291,8 @@ public class FieldConstants {
     public static final String FILE_CHECKSUM = "The checksum of the file.";
     public static final String FILE_PATH = "The path of the file.";
     public static final String FILE_URI = "The uri of the file.";
-    public static final String FILE_EXTERNAL = "Indicates the file is external or not.";
+    public static final String FILE_EXTERNAL = "Indicates whether the file comes from an external path or not.";
+    public static final String FILE_RESOURCE = "Indicates the file is treated as a resource.";
```

Comment:
Isn't this param almost identical to the ParamConstant "FILE_RESOURCE_DESCRIPTION" ? Why?